interfaces/cython/cantera/utils.pyx

```diff
@@ -76,6 +77,9 @@ cdef public PyObject* pyCanteraError = <PyObject*>CanteraError
 cdef anyvalue_to_python(string name, CxxAnyValue& v):
     cdef CxxAnyMap a
     cdef CxxAnyValue b
+    if v.isType[void]():
```

Comment:
I had introduced `AnyValue::isEmpty` here.

Reply:
You can drop that removal if you like (the C++ version is `v.is<void>()`, which looks slightly less weird, but that clashes with `is` being a keyword in Python). I don't think it necessarily needs to be exposed in Python, though. I'm hoping these conversion functions will pretty much be the boundary for the `AnyMap`/`AnyValue` classes in Python.

Reply:
I'd be :+1: if we could keep `isEmpty` (and also use it here) as I was hunting for this before in the context of `setParameters` within C++, and `v.is<void>()` definitely didn't come to mind when I was looking over the header file :wink:

Reply:
Regarding the boundary for the Python API, I believe getting access to the unit system may be good, but there isn't a pressing need at the moment.

Reply:
Please go ahead and drop the last commit on this branch, then.

Reply:
done.